Dockerfile

```diff
@@ -18,7 +24,17 @@ RUN pipenv install --dev --system --deploy --ignore-pipfile
 COPY ./package.json /app/package.json
 COPY ./yarn.lock /app/yarn.lock
 
-# Actually run things:
+COPY . /app
```

Comment:
Can you clarify why this is added? Does it mean the above lines (copying package.json and yarn.lock) are no longer needed? Does it copy _everything_ in the local directory, including ignored files/directories?